jepsen/project.clj

```diff
@@ -17,6 +17,7 @@
                   :exclusions [org.slf4j/slf4j-api
                                org.slf4j-log4j12
                                com.google.guava/guava]]]
+  :classifiers [["rabbitmq" :rabbitmq]]
```

Comment:
Ah, yeah, that's still buried inside jepsen isn't it. I gotta pull that out into its own project. Classifier sounds good for now. :)